src/ray/stats/metric_exporter_client.cc

```diff
@@ -54,18 +54,28 @@ void MetricsAgentExporter::ReportMetrics(const std::vector<MetricPoint> &points)
   MetricExporterDecorator::ReportMetrics(points);
   rpc::ReportMetricsRequest request;
   for (auto point : points) {
-    auto metric_point = request.add_metrics_point();
+    auto metric_point = request.add_metrics_points();
     metric_point->set_metric_name(point.metric_name);
     metric_point->set_timestamp(point.timestamp);
     metric_point->set_value(point.value);
     auto mutable_tags = metric_point->mutable_tags();
     for (auto &tag : point.tags) {
       (*mutable_tags)[tag.first] = tag.second;
     }
+    // If description and units information is requested from
+    // the metrics agent, append the information.
+    if (should_update_description_) {
+      metric_point->set_description(point.measure_descriptor.description());
+      metric_point->set_units(point.measure_descriptor.units());
+    }
```

Comment:
Does this mean we'll send the description for _all_ defined metrics each time we define a new one? That's probably ok, but ideally we would only update the necessary ones

Reply:
Yes that's correct. I chose to this because it is simple and adding new metrics should be extremely rare (usually they all happens once when the process starts).

Reply:
I can add a TODO here.